srcpkgs/lazarus-lcl-gtk2/template

```diff
@@ -0,0 +1,37 @@
+# Template file for 'lazarus-lcl-gtk2'
+pkgname=lazarus-lcl-gtk2
+version=1.8.4
+revision=1
+wrksrc=lazarus
+build_style=gnu-makefile
+makedepends="fpc gtk+-devel gdk-pixbuf-devel"
+depends="lazarus"
+short_desc="LCL units for gtk2"
+maintainer="mobinmob <mobinmob@disroot.org>"
+license="modified LGPL-2"
+homepage="https://www.lazarus-ide.org/"
+distfiles="https://sourceforge.net/projects/lazarus/files/Lazarus%20Zip%20_%20GZip/Lazarus%201.8.4/lazarus-1.8.4.tar.gz"
+checksum=5a9801ac341a322ad1fa104d373f154d8ab72ba0add1693d0dcc8d936e6c10e9
+nopie=yes
```

Comment:
add space between variables and functions

Reply:
Fixed on all 4 templates.